modules/extensions/src/views/lite/components/debugger/style.scss

```diff
@@ -124,7 +124,7 @@
     li {
       position: relative;
 
-      & > label {
```

Comment:
you sure? I was really trying to target the <label> element (.label)'s parent @allardy

Reply:
That seems to be good, were you trying something else ?  The unset felt wrong 

![image](https://user-images.githubusercontent.com/42552874/64887156-f72c5800-d635-11e9-9af3-e8412cef6344.png)